fix: Language review and Vale rules update

themr0c opened the pull request:
Adapt vale configuration:

* Skip validation on metadata (`:attribute:` and `id` statements).
* Use the  new `CheDocs.TechnicalTerms` rule  to define capitalization rules, with severity `error`, instead of the Vocab (`accept.txt`, `reject.txt`).
* Use the Vocab (`accept.txt`, `reject.txt`) only to expand the dictionary (words triggering `Spelling` errors).
* Raise the severity of the `CheDocs.Attributes` substitution rule.

Fix the content.

themr0c committed:
Vale configuration raising severity of capitalization and attributes rules

themr0c committed:
Fix language on modules/administration-guide/partials/con_che-workspaces-architecture.adoc

themr0c committed:
Fix language on modules/administration-guide/partials/proc_configuring-bitbucket-server-oauth1.adoc

themr0c committed:
Fix language on modules/administration-guide/partials/ref_grafana-dashboards-for-che.adoc

themr0c committed:
Fix language on modules/contributor-guide/partials/ref_che-theia-plug-in-api.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/assembly_using-go-artifact-repositories.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/proc_using-self-signed-certificates-in-go-projects.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/proc_creating-a-workspace-from-the-default-branch-of-a-git-repository.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/proc_changing-the-configuration-of-an-existing-workspace.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/proc_using-self-signed-certificates-in-nuget-projects.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/proc_using-self-signed-certificates-in-maven-projects.adoc

yhontyk commented on .vale/styles/Vocab/Che/accept.txt:
```diff
@@ -37,7 +37,6 @@ AsciiDoc
 autostart
 AWS|aws
 Bitbucket
-boolean
```
Why do we remove this? For different purposes, Boolean and boolean are accepted.

themr0c replied:
Indeed. We have `[Bb]oolean` in the accept.txt (line 2) to add it to the dictionary (`Spelling` rules) and accept both forms. The line containing `boolean` is therefore obsolete.

yhontyk replied:
Ah! Good! Sorry for missing it

yhontyk commented on .vale/styles/Vocab/Che/reject.txt:
```diff
@@ -12,8 +13,6 @@ now
 our
 out of the box
-[Pp]lease
-pod
```
`pod` will be reported by Vale as an unknown work AFAIK.

themr0c replied:
This is a change in the `reject.txt` file which was refusing `pod`. The scoping for the vocabulary is not perfect, so for capitalization, the job is moving to the `TechnicalTerms` rule (with a more complete error message).

yhontyk replied:
Ack, makes sense

themr0c committed:
Vale configuration raising severity of capitalization and attributes rules

themr0c committed:
Vale configuration raising severity of capitalization and attributes rules

themr0c committed:
New attribute

themr0c committed:
Fix language on modules/end-user-guide/partials/assembly_viewing-logs-for-dotnet-with-omnisharp-theia-plug-in.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/proc_provisioning-jetbrains-activation-code-for-offline-use.adoc

themr0c committed:
Fix language on modules/installation-guide/partials/proc_sizing-google-cloud-node-pools-for-your-eclipse-che-workspaces.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/proc_using-self-signed-certificates-in-python-projects.adoc

themr0c committed:
Fix language on modules/installation-guide/partials/proc_preparing-the-aws-system-for-installing-che.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/proc_configuring_gitlab_authentication.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/proc_using-self-signed-certificates-in-gradle-projects.adoc

themr0c committed:
Fix language on modules/installation-guide/partials/proc_importing-untrusted-tls-certificates.adoc

themr0c committed:
Fix language on modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc

themr0c committed:
Fix language on modules/administration-guide/partials/proc_configuring-openshift-oauth-with-initial-user.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/proc_using-npm-artifact-repositories.adoc

themr0c committed:
Fix language on modules/installation-guide/partials/proc_upgrading-che-and-losing-user-data.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/proc_creating-a-workspace-from-the-default-branch-of-a-git-repository.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/proc_allowing-users-to-define-workspace-deployment-labels-and-annotations.adoc

themr0c committed:
Fix language on modules/installation-guide/examples/checluster-properties.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/proc_configuring_bitbucket_authentication.adoc

themr0c committed:
Fix language on modules/installation-guide/partials/proc_importing-untrusted-tls-certificates-old.adoc

themr0c committed:
Fix language on modules/installation-guide/partials/assembly_upgrading-che-namespace-strategies-other-than-per-user.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/ref_netcoredebugoutput-plug-in.adoc

themr0c committed:
Fix language on modules/administration-guide/partials/proc_configuring-che-to-use-external-identity-provider.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/assembly_viewing-logs-for-dotnet-with-netcoredebugoutput-plug-in.adoc

themr0c committed:
Fix language on modules/installation-guide/partials/proc_configuring-workspace-exposure-strategies-using-a-helm-chart.adoc

themr0c committed:
Fix language on modules/installation-guide/partials/proc_preparing-the-aws-system-for-installing-che.adoc

themr0c committed:
Fix language on modules/end-user-guide/examples/snip_gitlab-personal-access-token-secret.adoc

themr0c committed:
Fix language on modules/end-user-guide/partials/proc_configuring_gitlab_authentication.adoc

themr0c committed:
Fix language on modules/installation-guide/partials/assembly_configuring-che-on-google-cloud-platform.adoc

themr0c committed:
Fix fixtures to avoid triggering errors in the GitHub workflow

yhontyk commented on modules/installation-guide/partials/proc_preparing-the-aws-system-for-installing-che.adoc:
```diff
@@ -4,6 +4,8 @@
 The following sections describe how to Configure {kubernetes} for {prod}
 on Amazon Elastic Compute Cloud (Amazon EC2).
 
+boolean Boolean
```
This should not be here

themr0c replied:
```suggestion
```

OMG. a debug leftover oO.

yhontyk commented on .vale/styles/CheDocs/TechnicalTerms.yml:
```diff
@@ -0,0 +1,29 @@
+  '\sjava': Java
+  '\netcoredebugoutput': NetcoredebugOutput
+  '\soperator': Operator
+  '\spod': Pod
```
Do we capitalize it to match Kubernetes' documentation?
OpenShift has given up the k8s-style capitalization (they only capitalize Operators now).

themr0c replied:
I guess so.

yhontyk commented on .vale/styles/CheDocs/TechnicalTerms.yml:
```diff
@@ -0,0 +1,29 @@
+  '\sContainer Registry': Red Hat Ecosystem Catalog
+  '\sContainer Catalog': Red Hat Ecosystem Catalog
+  '\sconfig map': ConfigMap
+  '\sconfig maps': ConfigMaps
```
This is also according to k8s docs style but not OpenShift. OpenShift docs do not pluralize API objects https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#object-references

I'm mostly bringing this up so we as a team decide clearly on the style we want to follow and enforce in the docs.

skabashnyuk commented on modules/installation-guide/partials/proc_preparing-the-aws-system-for-installing-che.adoc:
```diff
@@ -166,7 +168,7 @@ Output similar to the following means a needed role must be created manually:
 [subs="+quotes",options="nowrap"]
 ----
-arn:aws:sts::269287474311:assumed-role...4bff is not authorized to perform: iam:CreateServiceLinkedRole on resource: arn:aws:iam::269287474311:role/aws-service-role/elasticloadbalancing.amazonaws.com/AWSServiceRoleForElasticLoadBalancing
+arn:aws:`ts::269287474311:assumed-role...4bff is not authorized to perform: iam:CreateServiceLinkedRole on resource: arn:aws:iam::269287474311:role/aws-service-role/elasticloadbalancing.amazonaws.com/AWSServiceRoleForElasticLoadBalancing
```
is there a reason for this change?

themr0c replied:
```suggestion
arn:aws:sts::269287474311:assumed-role...4bff is not authorized to perform: iam:CreateServiceLinkedRole on resource: arn:aws:iam::269287474311:role/aws-service-role/elasticloadbalancing.amazonaws.com/AWSServiceRoleForElasticLoadBalancing
```

typo

themr0c committed:
Update modules/installation-guide/partials/proc_preparing-the-aws-system-for-installing-che.adoc

themr0c committed:
Fix typo

themr0c committed:
Merge branch 'master' into vale-styles-request-attributes

themr0c committed:
Set ExternalTimeout: 60 # (seconds) default is 15.
for  https://img.shields.io/github/issues-raw/eclipse/che-che4z-explorer-for-endevor?style=flat-square false positive
